designs/webviews/webviews-general.md

```diff
@@ -0,0 +1,95 @@
+# Webviews for Rich UIs in VS Code
+
+## Objective
+
+Justify expanding the criteria for using webviews within the AWS Toolkit for VS Code in order to create richer UIs than VS Code natively allows.
+
+### Out-of-scope
+
+* Architectures or frameworks for the webviews
+  * A best-practices guide to developing webviews with the architecture/framework decided on above.
+* Look and feel discussions
+* A detailed list of user experiences to port to webviews
+
+## Tenets (unless you know better ones)
+
+1. Users should have a delightful experience when interacting with AWS features through our AWS Toolkit.
+2. We should not limit ourselves to IDE-specific conventions if it does not provide the best user experience possible.
```

Comment:
I think we need to make a note that we want to preserve an idiomatic VS code experience as much as possible - whilst at the same time enriching it. (e.g. decent keyboard-only experience, minimalistic, etc)

Reply:
Treat documentation as neutral/factual. Please remove all "we/our/I" notation.